pkg/resource/notebook_instance/custom_set_input_update.go

```diff
@@ -0,0 +1,28 @@
+package notebook_instance
+
+import (
+	ackcompare "github.com/aws-controllers-k8s/runtime/pkg/compare"
+	"github.com/aws/aws-sdk-go/aws"
+	svcsdk "github.com/aws/aws-sdk-go/service/sagemaker"
+)
+
+// handleUpdateOnlyParameters sets Disassociate<field> to true if corresponding value in desired is nil(ie not included in the spec)
+// and if the corresponding value in the latest spec is not nil (ie contains some value)
+// Ex if LifecycleConfigName was "a" in the latest spec but nil in the desired spec, DisassociateLifecycleConfig is set to true
```

Comment:
This one is a good example of a descriptive comment on a function.

Reply:
Thanks! :)

Reply:
good